lib/utils.js

```diff
@@ -33,6 +34,47 @@ exports.basicAuth = function(username, password) {
 	};
 };
 
+// Matches routes
+exports.matchRoutes = function(req, res) {
+
+  var path = (req.params[0]);
+  res.render(path, function(err, html) {
+    if (err) {
+      res.render(path + "/index", function(err2, html) {
+        if (err2) {
+          console.log(err);
+          res.status(404).send(err + "<br>" + err2);
+        } else {
+          res.end(html);
+        }
+      });
+    } else {
+      res.end(html);
+    }
+  });
+
+};
+
+// Synchronously get the url for the latest release on github
+exports.getLatestRelease = function() {
+  var url = "";
+  var options = {
+    headers: {'user-agent': 'node.js'}
+  }; 
+  var gitHubUrl = 'https://api.github.com/repos/alphagov/govuk_prototype_kit/releases/latest';
+  try {
+      res = request('GET', gitHubUrl, options);
```

Comment:
I'm not sure this is valid code - request is normally async. When I try it locally, and add a console.log(err), I get:

```
{ [Error: Server responded with status code 403:
{"message":"API rate limit exceeded for 80.194.77.100. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)","documentation_url":"https://developer.github.com/v3/#rate-limiting"}]
  statusCode: 403,
  headers: 
   { server: 'GitHub.com',
     date: 'Thu, 25 Feb 2016 17:19:41 GMT',
     'content-type': 'application/json; charset=utf-8',
     'transfer-encoding': 'chunked',
     connection: 'close',
     status: '403 Forbidden',
     'x-ratelimit-limit': '60',
     'x-ratelimit-remaining': '0',
     'x-ratelimit-reset': '1456421018',
     'x-github-media-type': 'github.v3; format=json',
     'access-control-allow-credentials': 'true',
     'access-control-expose-headers': 'ETag, Link, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval',
     'access-control-allow-origin': '*',
     'content-security-policy': 'default-src \'none\'',
     'strict-transport-security': 'max-age=31536000; includeSubdomains; preload',
     'x-content-type-options': 'nosniff',
     'x-frame-options': 'deny',
     'x-xss-protection': '1; mode=block',
     'x-github-request-id': '50C24D64:1678C:5C7714:56CF37AD' },
  body: <Buffer 7b 22 6d 65 73 73 61 67 65 22 3a 22 41 50 49 20 72 61 74 65 20 6c 69 6d 69 74 20 65 78 63 65 65 64 65 64 20 66 6f 72 20 38 30 2e 31 39 34 2e 37 37 2e ... >,
  url: 'https://api.github.com/repos/alphagov/govuk_prototype_kit/releases/latest' }
```